public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
dont commit dist shit

Reply:
thats not dist shit, thats the template used for building the dist shit